### PR TITLE
Add wetix logo and svelte-reactive-form

### PR DESCRIPTION
**Pull request by si3nloong**
Add [WeTix](https://github.com/wetix) and [`svelte-reactive-form`](https://github.com/wetix/svelte-reactive-form)

### CODE REVIEW

**Comment by swyxio** [comment]
Deploy Preview is done!

Built with commit f39aabf6c6a8f8b3bfba83e0e723f384a71b8ae2

https://deploy-preview-352--svelte-community.netlify.app

**Comment by si3nloong** [comment]
Hi, any idea when is this going to merge?

File: whos-using-svelte/WhosUsingSvelte.svelte [hunk 1]
```diff
@@ -201,6 +201,7 @@
 	<a target="_blank" rel="noopener" href="https://seagage.com"><img src="https://static.wixstatic.com/media/3e1d33_2e8f3c427913446ea9091cdc903cd4fb~mv2.png/v1/fill/w_192,h_38,al_c,q_85,usm_0.66_1.00_0.01/3e1d33_2e8f3c427913446ea9091cdc903cd4fb~mv2.webp" alt="Seagage Inc" loading="lazy"/></a>
 	<a target="_blank" rel="noopener" href="https://www.creative-tim.com/templates/svelte?ref=svelte.dev"><img src="organisations/ct-logo-text-black.png" alt="Creative Tim" loading="lazy"/></a>
 	<a target="_blank" rel="noopener" href="https://turgensec.com"><img src="organisations/TurgenSec.png" alt="TurgenSec" loading="lazy"/></a>
+	<a target="_blank" rel="noopener" href="https://wetix.my" class="WeTix"><img src="organisations/wetix.svg" alt="WeTix" loading="lazy"/></a>
```
Comment (dreitzner) [inline whos-using-svelte/WhosUsingSvelte.svelte]: why use a class `.wetix` in `a`-tag

**Review by dreitzner — CHANGES_REQUESTED** [review]
- [x] remove unused css class

everything else seems fine

File: whos-using-svelte/WhosUsingSvelte.svelte (hunk 1 above)
Comment (si3nloong) [reply whos-using-svelte/WhosUsingSvelte.svelte]: done the fix. Can we deploy it very soon?

**Review by dreitzner — CHANGES_REQUESTED** [review]
@si3nloong please pull in the latest changes from master and add your info to https://github.com/sveltejs/community/blob/master/whos-using-svelte/WhosUsingSvelte.js

File: data/code.yml [hunk 2]
```diff
@@ -2464,3 +2464,13 @@ resources:
     stars: 1000
     last_updated: '2020-09-08T15:28:37Z'
     downloads: 1941
+  - name: '`svelte-reactive-form`'
```
Comment (benmccann) [inline data/code.yml:2467]: components now live at https://github.com/svelte-society/sveltesociety.dev

**Pull request closed without merging**